### ISSUE TITLE
multi agent trajectory prediction?

### ISSUE THREAD

**Issue by pjw1** [uber-research/LaneGCN#9]
Thank you for sharing your work and neat code!

In your paper (3.5 Learning),
the loss is used for training multiple agent. (M is the number of agent in the scene)

But the pretrained checkpoint (36.000.ckpt) seems only predicting the first agent for computing scores such as minADE. 

Do you also have the weight checkpoint for multi agent?
If so, can you share one?
Training process is too heavy for my server ;(

**Comment by pjw1** [comment]
Sorry, I just confused. 
Given weight (36.000) is trained for multiple agent.

**Comment by pjw1** [comment]
Sorry. I just confused.

The checkpoint and the train code is for training multiple agent.
